src/dynapi/SDL_dynapi.sym

```diff
@@ -94,7 +94,7 @@ SDL3_0.0.0 {
     SDL_CreateGPUTexture;
     SDL_CreateGPUTransferBuffer;
     SDL_CreateHapticEffect;
-    SDL_CreateIOAsyncGroup;
+    SDL_CreateIOAsyncQueue;
```

Comment:
Can we change IOAsync to AsyncIO?

Reply:
Yes, I was just thinking the same thing. This started out as a compliment to SDL_IOStream, but this is it's own thing now, for sure.

Reply:
Fixed in 9f038d1cedf3ea9a68e962ec572a49f79b7c2bd2.